src/utils/logging.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  getApplicationName,
+  getBundleId,
+  getDeviceName,
+  getSystemName,
+  getSystemVersion,
+  getVersion,
+  isEmulator,
+} from 'react-native-device-info';
+
+type AttributeMap = Record<string, string>;
+
+export const logger = (() => {
+  const ddTok = 'pubb96b84a13912504f4354f2d794ea4fab';
```

Comment:
i know this is a public token but is it fine hardcoding it like this?

Reply:
That's a very good question, I just ported what we did in [ios](https://github.com/Basis-Theory/basistheory-ios/blob/cef523f4ea59f1d208bc968a40fcc3c6581136c1/BasisTheoryElements/Sources/BasisTheoryElements/TelemetryLogging.swift#L14) but we should add them to our secrets

Reply:
That's a very good question, I just ported what we did in [ios](https://github.com/Basis-Theory/basistheory-ios/blob/cef523f4ea59f1d208bc968a40fcc3c6581136c1/BasisTheoryElements/Sources/BasisTheoryElements/TelemetryLogging.swift#L14) but we should add them to our secrets at some point(?)